app/controllers/annotation_categories_controller.rb

```diff
@@ -204,6 +204,11 @@ def upload
     redirect_to assignment_annotation_categories_path(assignment_id: @assignment.id)
   end
 
+  def get_annotation_text_stats
+    annotation_stats = AnnotationText.find_by(id: params[:annotation_text_id]).get_stats
```

Comment:
Use `find` instead of `find_by`. You can do this when you're searching by `id`, and `find` will raise an error if the record isn't found.